### ISSUE TITLE
Pre-commit support is broken

### ISSUE THREAD

**Issue by regit** [KeisukeYamashita/commitlint-rs#428]
**Describe the bug**

Pre-commit wants to run a cargo install of the project but it is failing to build it.

This is linked to the fact, the Cargo.toml has been switched to workspace.

## Config

If you have any config file, please paste it below:

I got a `.pre-commit-config.yaml ` with:

```
-   repo: https://github.com/KeisukeYamashita/commitlint-rs
    rev: main
    hooks:
    -   id: commitlint
```

## Error message

Please paste the error message below:

```
An unexpected error has occurred: CalledProcessError: command: ('/nix/store/8vpg72ik2kgxfj05lc56hkqrdrfl8xi9-bash-5.2p37/bin/bash', '-e', '/home/eric/.nix-profile/bin/cargo', 'install', '--bins', '--root', '/home/eric/.cache/pre-commit/repono3li81d/rustenv-system', '--path', '.')
return code: 101
stdout: (none)
stderr:
    error: found a virtual manifest at `/home/eric/.cache/pre-commit/repono3li81d/Cargo.toml` instead of a package manifest
Check the log at /home/eric/.cache/pre-commit/pre-commit.log
```

## Expected behavior

build is successful.

## Version

not yet available

## Additional context

This may be a pre-commit issue.
